l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -384,15 +385,9 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
         return _chainId;
     }
 
-    /// @notice This internal function is used to register a new zkChain in the system.
-    /// @param _chainId The chain ID of the ZK chain
-    /// @param _zkChain The address of the ZK chain's DiamondProxy contract.
-    /// @param _checkMaxNumberOfZKChains Whether to check that the limit for the number
-    /// of chains has not been crossed.
-    /// @dev Providing `_checkMaxNumberOfZKChains = false` may be preferable in cases
-    /// where we want to guarantee that a chain can be added. These include:
-    /// - Migration of a chain from the mapping in the old CTM
-    /// - Migration of a chain to a new settlement layer
+    /// @notice This function is used to register a new zkChain in the system.
+    /// @notice see external counterpart for full natspec.
+
```

Comment:
not sure why the comment was updated here from more infromative to less informative